my-app/src/pages/boardPage/boardPage.tsx

```diff
@@ -45,11 +55,50 @@ const BoardPage = () => {
         setIsCreate(true);
     }
 
+    function dragStartHandler(e: React.DragEvent<HTMLDivElement>, column: ColumnItemType): void {
+        setCurrentColumn(column);
+    }
+
+    function dragOverHandler(e: React.DragEvent<HTMLDivElement>): void {
+        e.preventDefault();
+       (e.target as HTMLDivElement).style.background = '#ece6e6'
+    }
+
+    function dragEndHandler(e: React.DragEvent<HTMLDivElement>): void {
+     (e.target as HTMLDivElement).style.background = ''
+    }
+
+    function dropHandler(e: React.DragEvent<HTMLDivElement>, column: ColumnItemType): void {
+        e.preventDefault();
+       dispatch(
+        sortColumnList([...columnList], column, currentColumn)
+       );
+      (e.target as HTMLDivElement).style.background = ''
+    }
+
+    const sortColumns = (a: ColumnItemType, b: ColumnItemType) => {
+      if(a.columnOrder > b.columnOrder) {
```

Comment:
i think something like this will also work:
``` 
 const sortColumns = (column1: ColumnItemType, column2: ColumnItemType) => a.columnOrder - b.columnOrder; 
```

And please do not use `a,b`, use something more specific like `column1, column2`